.github/workflows/presubmit.yaml

```diff
@@ -0,0 +1,42 @@
+name: presubmit
+on: [push]
+jobs:
+  windows-test:
```

Comment:
Should this be named `presubmit-windows`? Is this essentially the same as the presubmit bot below?

Reply:
The presubmit jobs do version-specific builds. This one runs unit tests on Windows. It is more similar to the checker job but I want to have "windows" in the name. I think this name is descriptive.